docs/conf.py

```diff
@@ -96,8 +102,8 @@
 
 
 def setup(app: Sphinx):
-    # Don’t allow broken links. DO NOT CHANGE THIS LINE, fix problems instead.
-    app.warningiserror = True
```

Comment:
By now, this is enforced in .readthedocs.yml

Having it twice helped against the most egregious offenders, but by now I’m in a more active role and can prevent the other instance from being removed.